pkg/storage/unified/sql/test/integration_test.go

```diff
@@ -349,7 +349,7 @@ func TestIntegrationBackendList(t *testing.T) {
 		continueToken, err = sql.GetContinueToken(res.NextPageToken)
 		require.NoError(t, err)
 		require.Equal(t, rv8, continueToken.ResourceVersion)
-		require.Equal(t, int64(4), continueToken.StartOffset)
+		require.Equal(t, int64(5), continueToken.StartOffset)
```

Comment:
I am surprised by StartOffset being 5 here 🤔 . We started with an offset of 2, listed 2 items, so next page should offset the first 4 results.  I also can't spot where this is changing ? What am I missing ?

Reply:
Good point, it shouldn't have been changed. The issue was related to the <= bug, because I switched the token creation to after iter.Next(), the token was off. I changed it back to creating the token before the iter.Next() and this is back to normal.